go.mod

```diff
@@ -165,6 +165,7 @@ require (
 	github.com/gorilla/mux v1.8.0 // indirect
 	github.com/gosuri/uitable v0.0.4 // indirect
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0 // indirect
```

Comment:
We are in the frozen dependencies period. As this is introducing a new dependency, we won't be able to merge this PR by now.

Reply:
it's a change in test code, not production file. Surely that qualifies as an exception

Reply:
Technically, whereas we should avoid adding new deps in the code, we postponed the deadline to October, so if this PR is really needed for the release, we could re-trigger the process if we all agree.

Reply:
Understood. This PR is not urgent and it can wait. I have other things I will work on in the meantime

Reply:
@gfichtenholt , please feel free to merge the PR whenever you want. The release is finally out!